core/src/main/resources/lib/form/dismiss.jelly

```diff
@@ -0,0 +1,48 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi, Yahoo! Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="jelly:stapler" xmlns:l="/lib/layout">
+  <s:documentation> <![CDATA[
+    Dismiss button themed by YUI. This should be always
+    used instead of a custom button tag.
+    ]]>
+    <s:attribute name="name">
+      If specified, becomes the value of the name attribute.
+      When you have more than one dismiss button on the form, this can be used to determine
+      which button is pressed, as the server will get a parameter by this name.
+    </s:attribute>
+    <s:attribute name="tooltip" use="required">
+      The tooltip text of the dismiss button. Something like "dismiss", etc.
+    </s:attribute>
+  </s:documentation>
+  <button
+    name="${attrs.name ?: 'Dismiss'}"
+    tooltip="${attrs.tooltip ?: '%Dismiss'}"
+    class="jenkins-button jenkins-button--tertiary"
+    type="submit">
+    <l:icon src="symbol-close" />
+  </button>
+
+</j:jelly>
```

Comment:
I would prefer to use the name `dismiss-button` as this jelly is providing a button and not a dismiss dialog.